package/yast2-network.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Tue Aug  9 11:18:28 UTC 2016 - ancor@suse.com
+
+- storage-ng: commented remaining code relying on the old
+  yast2-storage
+- Removed yast2-storage dependency
+- 3.2.2
+
```

Comment:
That's only checked if it's submitted to Factory or SLE. So far we are not submitting anywhere, just committing to our own project.